enginetest/queries/information_schema_queries.go

```diff
@@ -1451,6 +1451,29 @@ from information_schema.routines where routine_schema = 'mydb' and routine_type
 			},
 		},
 	},
+	{
+		SkipPrepared: true,
+		Name:         "correct schema for a recursive cte that calculate fibonacci numbers",
```

Comment:
(minor) might be good to mention the type promotion bug more directly just to help us remember why we originally wrote this. Speaking of... not required, but you could also drop in a comment with a link to the associated bug this one was testing.

Reply:
This test seems out of place in the `information_schema_queries` file. I took a quick look and didn't see recursive CTE tests grouped together anywhere, so maybe consider moving this to `QueryTests` in `queries.go`? If you can find a more specific spot, that would be even better.

Reply:
Will add a link to the original bug.

I think this is the correct spot for this test, since other tests are doing all sorts of column checks here.
But there's a [bug](https://github.com/dolthub/dolt/issues/3168) that is blocking us from doing the same kind of column check that's already happening here: the view column data isn't available to us, so I need to use `desc cte_view` to get column info. After bug 3168 is fixed, we can update this test to consult `information_schema.columns` just like other tests in this file.